docs/src/pages/demo.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+export default function Demo() {
+  return (
+    <>
+      <iframe
+        src={useBaseUrl("/try.html")}
```

Comment:
How come this was named `try`? Seems odd...

Reply:
Try as in "try the demo"  😅 no the best name